blacklight-cornell/app/assets/stylesheets/cornell/_facets.scss

```diff
@@ -39,7 +39,9 @@
 	}
 }
 .facets-heading {
-	font-size: 1.25rem;
+	font-size: 1.25rem
```

Comment:
should this have a semicolon at end? also can the 2 lines of commented out css be cleaned out, or is that being saved for later?

Reply:
yes, it should, for consistency!